File: cpp/bc1187.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main()
{
	double vec[12][12], sum = 0.0;
	int aux = 10, count = 1;
	char op;
	
	cin >> op;
	
	for(int i = 0; i < 12; i++)
	{
		for(int j = 0; j < 12; j++)
			cin >> vec[i][j];
	}
	
	for(int i = 0; i < 12; i++)
	{
		for(int j = count; j <= aux; j++)
			sum += vec[i][j];
		
		
		aux -= 1;
		count++;
	}
	
	if(op != 'S') sum /= 30.0;
	
	cout << fixed << setprecision(1) << sum << endl;
}

File: python/bc1187.py
ma = [[None for j in range(12)] for i in range(12)]
res = 0.0
aux = 10
count = 1

op = input()

for c in range(0, 12):
    for d in range(0, 12):
        ma[c][d] = float(input())

for c in range(0, 12):
    for d in range(count, aux+1):
        res += ma[c][d];
    aux -= 1
    count += 1

if op != "S": res /= 30.0

print(f"{res:.1f}")